src/Enterspeed.Source.UmbracoCms/App_Plugins/Enterspeed.Dashboard/Dashboard.Sub.Controllers/selectNode.controller.js

```diff
@@ -44,13 +51,19 @@
     vm.submit = function () {
         if ($scope.model && $scope.model.submit) {
             if (vm.includeEverything) {
-                $scope.model.target.id = "-1";
-                $scope.model.target.name = "Everything";
-                $scope.model.target.icon = "icon-item-arrangement";
-                $scope.model.target.includeDescendents = true;
+                $scope.model.targets = [];
+                $scope.model.targets.push({
+                    id: "-1",
+                    name: "Everything",
+                    icon: "icon-item-arrangement",
+                    includeDescendants: true
+                });
 
             } else {
-                $scope.model.target.includeDescendents = vm.includeDescendents;
+                for (let i = 0; i < $scope.model.targets.length; i++) {
+                    
+                    $scope.model.targets[i].includeDescendants = vm.includeDescendants;
```

Comment:
Hi @jesperweber 

Include descendants should be set here. I am making sure that it set on all selected nodes for that specific session of the dialog. 

Is it not working for you?

Reply:
Hmm it does seems to be working now, maybe it was just a caching issues on my side?

Well it looks nice and is working :-)

Reply:
It was probably some browser caching :)